src/app.css

```diff
@@ -0,0 +1,2 @@
+.navLink { color: inherit !important; }
```

Comment:
Copyright header (probably want to adapt these to shorter spdx headers as well (just shorter, however need to keep GPL3 for now since it is the original before the copy)

Reply:
@axelchalon Can you add a copyright header to this file too?